glusterd2/commands/volumes/volume-option.go

```diff
@@ -185,6 +185,10 @@ func volumeOptionsHandler(w http.ResponseWriter, r *http.Request) {
 			DoFunc: "vol-option.Validate",
 			Nodes:  []uuid.UUID{gdctx.MyUUID},
 		},
+		{
```

Comment:
Remove the redundant one that's down below. Also, add undo function (that already exists).